src/core/apim.tf

```diff
@@ -572,6 +573,15 @@ resource "azurerm_api_management_custom_domain" "api_custom_domain" {
     )
   }
 
+  proxy {
+    host_name = local.prf_domain
+    key_vault_id = replace(
+      data.azurerm_key_vault_certificate.app_gw_platform.secret_id,
+      "/${data.azurerm_key_vault_certificate.app_gw_platform.version}",
```

Comment:
this certificate contains the common name `api.<env>.platform.pagopa.it`

Reply:
> `prf` is only for `UAT` env... So I think you need to use `count` and `dynamic` structures to prevent errors in `DEV` and `PROD` envs

I created a separate `azurerm_api_management_custom_domain` only in uat env , okay?

Reply:
@pasqualedevita is it okay to use dynamic to have it only in uat?